plugin.xml

```diff
@@ -33,6 +33,14 @@
             <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
         </config-file>
 
+        <config-file target="AndroidManifest.xml" parent="/*">
+            <uses-permission android:name="android.permission.BLUETOOTH" />
+        </config-file>
+
+        <config-file target="AndroidManifest.xml" parent="/*">
+            <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
+        </config-file>
+
```

Comment:
I believe these were intentionally left out with the intention that individual projects can add these if needed. @nickpatrick can you weigh in?